linkerd/protocol/h2/src/main/scala/io/buoyant/linkerd/protocol/h2/istio/IstioIdentifier.scala

```diff
@@ -0,0 +1,67 @@
+package io.buoyant.linkerd.protocol.h2.istio
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.twitter.finagle.buoyant.h2._
+import com.twitter.finagle.{Dtab, Path, Stack}
+import com.twitter.util.Future
+import io.buoyant.config.types.Port
+import io.buoyant.k8s.istio._
+import io.buoyant.k8s.istio.identifiers.InternalTrafficIdentifier
+import io.buoyant.k8s.istio.mixer.MixerClient
+import io.buoyant.linkerd.IdentifierInitializer
+import io.buoyant.linkerd.protocol.H2IdentifierConfig
+import io.buoyant.router.RoutingFactory.{BaseDtab, DstPrefix, Identifier, RequestIdentification}
+
+class IstioIdentifier(
+  pfx: Path,
+  baseDtab: () => Dtab,
+  routeCache: RouteCache,
+  clusterCache: ClusterCache,
+  mixerClient: MixerClient
+)
+  extends Identifier[Request] {
+  val istioIdentifier = new InternalTrafficIdentifier[Request](pfx, baseDtab, routeCache, clusterCache, mixerClient, new H2IstioRequestHandler)
+
+  override def apply(originalRequest: Request): Future[RequestIdentification[Request]] = {
+    val req = H2IstioRequest(originalRequest, None)
```

Comment:
:heart_eyes: explicit None

Reply:
this phrase gives me unpleasant [FORTRAN flashbacks](http://www.personal.psu.edu/jhm/f90/statements/implicit.html)